docs/src/routes/theme/index.js

```diff
@@ -0,0 +1,201 @@
+// Dependencies
+import { h, Component } from "preact";
+
+// Material Components
+import Button from "preact-material-components/Button";
+
+// Components
+import ComponentTable from "../../components/component-table";
+import CodeBlock from "../../components/code-block";
+
+// Styles
+import style from "./style";
+
+// Class
+export default class ThemePage extends Component {
+  render() {
+    return (
+      <div className={style["page-theme"]}>
+        <div className="mdc-typography--display1">Original documentation</div>
+        <div className="mdc-typography--body">
+          preact-material-components brings{" "}
+          <span className="strong">mdc-theme</span>as is, you can refer to its
+          documentation
+          <a href="https://github.com/material-components/material-components-web/tree/master/packages/mdc-theme">
+            {" "}
```

Comment:
Guess this should be after ```documentation``` two lines up.

Reply:
didnt get this